Gemfile

```diff
@@ -6,3 +6,5 @@ gemspec
 rails_version = "#{ENV['RAILS_VERSION'] || '6.0.0'}"
 
 gem "rails", rails_version == "master" ? { github: "rails/rails" } : rails_version
+
+gem "capybara", ">= 3.26"
```

Comment:
I think this dependency should be declared in the gemspec, since one of the gem's library files explicitly `require`s capybara:
https://github.com/github/actionview-component/blob/d64a6a6118573e86b40d3854be7edcf987cf7fdf/lib/action_view/component/test_helpers.rb#L3